src/stores/appcatalog/selectors.ts

```diff
@@ -32,10 +32,10 @@ export const selectApps = createSelector(
         allApps.push(
           ...Object.entries(catalog.apps).map(([appName, appVersions]) => ({
             name: appName,
+            appIconURL: appVersions[0].icon,
```

Comment:
This (`appVersions[0]`) is the latest version, right?

Reply:
It should be yeah. We could actually ensure that by sorting all the versions early in the action that fetches the catalog, but it should come with the latest at the first position.